### ISSUE TITLE
Assorted Planet Issues

### ISSUE THREAD

**Issue by Vakothu** [UristMcStation/UristMcStation#412]
- The mine has no air, breathable or not, despite being underground in a planet.
- ~~One of the random areas (The church-ish area) has a veil render, which last I checked straight-up summons Nar'sie.~~
- The Research Outpost's mass driver is totally useless on a planet.
- ~~The area where the above-ground mining outpost's disposal goes still has no air.~~
  - ~~In addition, all the rock around the mining outpost above-ground is impassible, which I assume is unminable, so if you get thrown down that disposal, RIP forever.~~
- ~~The description of the impassible rock walls is "Thick, long wet grass"~~
- ~~The description of the puddles is "A sink used for washing one's hands and face."~~

**Comment by Glloyd** [comment]
The mine having no air is intentional. It's in lieu of filling the mine
with hazardous gasses and incredibly hot air, which don't mesh well with
the airlock system and creates more issues for ZAS that airless turfs
don't. I just handwave it as NT didn't install a ventilation system, and
the mine's super deep so where a suit or die. But if it breaks peoples
immulsions that much, I can do something about it.

On Mon, Apr 4, 2016 at 12:25 AM, Michael <notifications@github.com> wrote:

> The mine has no air, breathable or not, despite being underground in a
> planet.
> One of the random areas (The church-ish area) has a veil render, which
> last I checked straight up summons Nar'sie.
> The description of the impassible rock walls is "Thick, long wet grass"
> The description of the puddles is "A sink used for washing one's hands and
> face."
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/UristMcStation/UristMcStation/issues/412>
>



-- 
Graham Lloyd

**Comment by Vakothu** [comment]
Did a quick update to the issues in this, nothing major, that's all I'll add to this issue before it gets too huge.

**Comment by Glloyd** [comment]
I totally forgot about that mass driver. I'll try to fix a bunch of these tomorrow. Also, feel free to keep adding, if stuff doesn't get posted here it won't get fixed.

**Comment by Glloyd** [comment]
Actually, I'll just point the mass driver towards the lake. Not like NT cares about polluting.

**Comment by scrdest** [comment]
The veil render in the cult chapel should be depowered if it isn't, same as the ones Wizard gets in his lair, I just didn't edit the desc to show it like the Wizard one did. I'll double-check.

The puddles' desc is inherited from BS12 code, I only noticed it after already having pushed the minimap.

**Comment by Vakothu** [comment]
Yeah, the Veil Render is depowered, also, while we're talking about the church, there's space under the windows.

**Comment by scrdest** [comment]
Fixed. Maple pointed it out to me.

**Comment by liction15** [comment]
the science outpost having only one hatchet makes it hard for multiple scientists to go do their own thing in the forest.

**Comment by Glloyd** [comment]
That's everything minus the air in the mine which I have a potential solution for. Any other issues that have come up?

**Comment by scrdest** [comment]
Building anything at all on the surface fucks with the atmos in strange ways; essentially, it reverts to default for, and only for, the affected area. That round where I accidentally made a hole to space, it drained air from the wooden floor that was built, but nothing else in the jungle.

Furthermore, the minimaps don't seem to handle light updates when placed. Mostly my bad, since I should have addressed that in my dynamic lighting PR.

**Comment by Glloyd** [comment]
Everything on here is fixed, except for the underground. I'm going to work on that now.

**Comment by CPU-Blanc** [comment]
> Everything on here is fixed, except for the underground. I'm going to work on that now.

@Glloyd  Ever fix the underground? :P

**Comment by Glloyd** [comment]
no :(
wouldn't be too difficult, but it's not on the priority list at the moment. i'll get to it as part of patching glloydstation up for the merge.
